src/main/resources/explanations/challenge29_reason.adoc

```diff
@@ -0,0 +1,5 @@
+*Why you should always clear your local storage once you have your work done in a website*
```

Comment:
Let's change this a bit: please have a look at the text & tone of the other reason docs. 
Next, best would be to use Session storage and/or cookies instead of localstorage.
If a secret does not need ot be known by the client, it is even better to encrypt the secret at the server side and send it. if it does not need ot be shared with other servers at other domains; maybe it is even better to just keep the secret at the server end.